velox/substrait/VeloxToSubstraitFunc.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/substrait/VeloxToSubstraitFunc.h"
+
+#include "velox/substrait/GlobalCommonVariable.h"
+
+namespace facebook::velox::substrait {
+
+uint64_t VeloxToSubstraitFuncConvertor::registerSubstraitFunction(
```

Comment:
@ZJie1 Thank you for iterating on this PR. I wonder what's the purpose of GlobalCommonVarSingleton and how will it work in a multi-threaded environment where multiple plan conversions will occur in parallel. It might be simpler to sync up of video chat. Feel free to ping me on Slack if you want to talk. Otherwise, we can continue on GitHub.

Reply:
Sure. Could you help me join into the velox slack by sending me the invite link for velox slack or other ways ? I have sent the email but haven't received the response...

Reply:
@jijufb Jiju, is this something you could help with?

Reply:
@ZJie1 Jiju will help with this sometime today.

Reply:
@ZJie1, can you share your email address to map to Velox Slack account to generate invite?

Reply:
I have given access to your intel account - jie1.zhang@intel.com access to Velox Slack. You should have received an invite. Let me know if you find any issues.